mllib/src/main/scala/org/apache/spark/mllib/feature/ChiSqSelector.scala

```diff
@@ -243,6 +245,19 @@ class ChiSqSelector @Since("2.1.0") () extends Serializable {
       case ChiSqSelector.FPR =>
         chiSqTestResult
           .filter { case (res, _) => res.pValue < alpha }
+      case ChiSqSelector.FDR =>
```

Comment:
Irrelevent to this PR, we can eliminate ```zipWithIndex``` at L235, since no one use it. Would you mind to do this clean up in your PR?